### PR TITLE
[libmesh] Update to latest release

### LINKED ISSUE #19933: [libmesh] update to 1.7.1
Library name: libmesh

New version number: 1.7.1

Other information that may be useful (release notes, etc...)
https://github.com/libMesh/libmesh/releases

### PR DESCRIPTION
**Pull request by PhoebeHui**
Fixes https://github.com/microsoft/vcpkg/issues/19933

The latest release version requires timpi.

### CODE REVIEW

**Review by JackBoosY — CHANGES_REQUESTED** [review]
```cmake
vcpkg_configure_make(
    SOURCE_PATH "${SOURCE_PATH}"
    OPTIONS ${OPTIONS}
)
```

File: ports/libmesh/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "libmesh",
-  "version-string": "1.5.0",
-  "port-version": 2,
+  "version-semver": "1.7.0-rc3",
```
Comment (JackBoosY) [inline ports/libmesh/vcpkg.json:3]: Should we use the rc version?

Comment (PhoebeHui) [reply ports/libmesh/vcpkg.json:3]: It's a pre-release revison, they may have final release version soon, we can wait for some days.

**Comment by autoantwort** [comment]
You can update to version 1.6.2

File: ports/libmesh/portfile.cmake [hunk 2]
```diff
@@ -1,11 +1,21 @@
...
 
+vcpkg_from_github(
+    OUT_SOURCE_PATH TIMPI_SOURCE_PATH
+    REPO libMesh/TIMPI
+    REF  2397962a0b2c4d19c53b1a45f0304eaea999e024
+    SHA512 b81c0b8ac03650c4de66e49355249100ccec47938cde11abd26535e0b6f2d35630f09b2510f041dc3104dffcb210f08790db870a04f8d0b02601c9d9d5e0c757
+    HEAD_REF master
+)
+
+file(COPY "${TIMPI_SOURCE_PATH}/." DESTINATION "${SOURCE_PATH}/contrib/timpi")
```
Comment (JackBoosY) [inline ports/libmesh/portfile.cmake:17]: Can you double confirm that the copied files has their paths?

**Comment by JackBoosY** [comment]
Closing this PR since it seems that no progress is being made. Please reopen if work is still being done.

**Pull request closed without merging**